src/background.js

```diff
@@ -1,6 +1,6 @@
 const REDIRECT_URL = browser.identity.getRedirectURL();
 const CLIENT_ID = 'c6d74070a481bc10';
-const SCOPES = ['profile keys'];
+const SCOPES = ['kinto profile keys'];
```

Comment:
I think what we are aiming for here with the encryption doc is: 

`const SCOPES = ['notes profile'];` or 
`const SCOPES = ['app:notes profile'];` or 
`const SCOPES = ['app:notes:write profile'];`

the FxA side of things would not know anything about 'kinto' scopes